content/posts/survey-launch/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "UC OSPO Network Launches Open Source Survey"
+date: 2025-04-16
+draft: false
+description: "The UC OSPO Network is conducting a survey to understand the open source practices and needs of the University of California community."
+tags: [network, survey]
+displayInList: true
+author: ["Virginia Scarlett"]
+
+resources:
+  - name: featuredImage
+    src: "firmbee-com-unsplash-small.jpg"
+    params:
+      description: "Image courtesy of firmbee.com via Unsplash"
+      showOnTop: true
+
+---
+*Image courtesy of firmbee.com via Unsplash*
+
+
+# Announcing the UC Open Source Survey 2025
```

Comment:
```suggestion
```

I know you couldn't preview this so you wouldn't know, but the title on line 2 will display like a Heading 1, so you don't need this one here.